### ISSUE TITLE
hardware filter to promiscuous mode

### ISSUE THREAD

**Issue by ehg1** [0xcafed00d/tracetcp#1]
I have installed the latest wincap library and attempted to use tracetcp and I get the following message
which I believe may be related to my wireless connection.

failed to set hardware filter to promiscuous mode

I was using the tracetcp as part of a batch file which allows me to simply input a couple of digits to instruct the program as to what server and port I wish to test.  The program returns that error msg.

If I try to execute tracetcp.exe all I get is a micro second view of what may be a cmd prompt screen and it is instantly closed and i receive no msg.

In researching the issue I found that those that are using wireshark which also uses wincap have had the same error.

MY OS is WIN XP sp3  and I am using Netgear WPN 311 wireless pci adapter w/ latest netgear driver installed.  

Is there a resolution to this problem?

thanks

ehg

**Comment by 0xcafed00d** [comment]
I have found that winpcap will not capture packets from WiFi adapters on Windows XP. However it generally does on windows 7 & and above (and  possibly vista but I have not tried it) 

Unfortunately this seems to be a limitation of XP so there is no work around possible in tracetcp. 

The only thing I can suggest is to use a later version of windows.

**Comment by ehg1** [comment]
Well I won't give up XP until absolutely forced too - Only gave up Win2000 Pro because I built a new box and the amd chipset did not support anything prior to xp!
 
I will try however to hardwire my box to my router/modem since my box also has a regular nic installed and see if that works
 
thanks for the timely reply
 
ehgOn Thu, 16 Apr 2015 15:37:56 -0700, Lee Witek wrote:>�I have found that winpcap will not capture packets from WiFi adapters on Windows XP. However it generally does on windows 7 &�and above (and possibly vista but I have not tried it)>>�Unfortunately this seems to be a limitation of XP so there is no work around possible in tracetcp.>>�The only thing I can suggest is to use a later version of windows.>>��>�Reply to this email directly or view it on GitHub.
